lib/miscParser.mli

```diff
@@ -17,7 +17,8 @@
 (** The basic types of architectures and semantics, just parsed *)
 
 (* Processor name with optional annotations *)
-type proc = int * string list option
+type func = Main | FaultHandler
```

Comment:
@maranget is this the right place to declare the func type? I was between miscParser and proc.mli.

Reply:
I have no strong opinion on this...